Source/gui/SNTNotificationManager.m

```diff
@@ -170,10 +190,38 @@ - (void)postDistributedNotification:(SNTMessageWindowController *)pendingMsg {
     kSigningChain : signingChain,
   };
 
-  [dc postNotificationName:@"com.google.santa.notification.blockedeexecution"
-                    object:@"com.google.santa"
-                  userInfo:userInfo
-        deliverImmediately:YES];
+  [self postDistributedNotificationWithUserInfo:userInfo
+                                  notificationName:@"com.google.santa.notification.blockedeexecution"];
+}
+
+- (void)postDistributedNotificationWithEvent:(SNTStoredEvent *)event withBundleHash: (NSString *) bundleHash {
```

Comment:
No spaces around the second parameters type

Reply:
fixed with `./Testing/fix.sh`